recipe/build.sh

```diff
@@ -29,16 +29,23 @@ EOF
 
 LINKFLAGS="${LINKFLAGS} -L${LIBRARY_PATH}"
 
-./bootstrap.sh \
-    --prefix="${PREFIX}" \
-    --without-libraries=python \
-    --with-toolset=${TOOLSET} \
-    --with-icu="${PREFIX}" \
-    || cat bootstrap.log
+if [[ "$CONDA_BUILD_CROSS_COMPILATION" == 1 ]]; then
+    CXX=$CXX_FOR_BUILD CC=$CC_FOR_BUILD ./bootstrap.sh \
```

Comment:
I'd rather we do this everywhere and remove the if condition. Maybe do `CXX=${CXX_FOR_BUILD:-${CXX}}`